pydotorg/urls.py

```diff
@@ -13,6 +13,7 @@
 
 handler404 = custom_404
 
+
```

Comment:
Style nit: Did you get a warning from your choice of linter or is this just an oversight?

Reply:
Oversight